cypress-tests/cypress/e2e/PaymentTest/00014-SaveCardFlow.cy.js

```diff
@@ -47,11 +47,7 @@ describe("Card - SaveCard payment flow test", () => {
           globalState
         );
         if (should_continue) {
-          // Don't continue if payment status is processing during auto capture
-          // Payment data is tokenized only after payment is successful
-          let notProcessing = res_data?.body?.status != "processing";
-          should_continue =
-            notProcessing && utils.should_continue_further(res_data);
+          should_continue = utils.should_continue_further(res_data);
```

Comment:
why was this reverted? this was added by @kashif-m to add support for WorldPay. Removing this would result in WorldPay tests failing.

Reply:
If status is processing after the payment confirm we cannot skip the next test cases because Psync call would make payment success and save card flow to work , we need to handle worldpay in a different way.